packages/owl-bot/test/create-pr.ts

```diff
@@ -97,6 +99,40 @@ describe('resplit', () => {
   });
 });
 
+describe('prependCommitMessage', () => {
+  for (const withCheckbox of [
+    WithRegenerateCheckbox.Yes,
+    WithRegenerateCheckbox.No,
+  ]) {
+    describe(`with checkbox: ${withCheckbox}`, () => {
+      it('handles an initial pull request content', () => {
+        const pullContent = resplit(
+          'feat: some feature\n\nadditional context',
+          withCheckbox
+        );
+        const prependedContent = prependCommitMessage(
+          'fix: some new feature\n\nmore additional context',
+          pullContent,
+          withCheckbox
+        );
+        snapshot(prependedContent.title);
```

Comment:
snapshot is hard to follow.  I'm much rather see two strings compared in line.

Reply:
The snapshots are easier to construct code-wise at the expense of the asserted value living in a different file. It is useful for large text bodies that are harder to read inline. I can switch this back to inline assertions though.

Reply:
Thank you for offering to switch back to inline assertions.  Please do.